src/datepicker/types.js

```diff
@@ -270,6 +270,11 @@ export type TimezonePickerPropsT = {
    * exact specificity, provide a date. Otherwise it will default to the relevant timezone at render.
    */
   date?: Date,
+  /**
+   * Customize the option's label. Useful for translations and optionally mapping from
+   * 'America/Los_Angeles' to 'Pacific Time'.
+   */
+  mapLabels?: OptionT => string,
```

Comment:
Any reason why to pass the whole `OptionT` and not just the label? Also, the select's label is type `React.Node`?

Reply:
good point about `React.Node` will update. 

in this case, the `id` is the IANA title (`America/Los_Angeles`), and the label contains some additional information like the offset, and short code (`PST`/`PDT`). I thought that extra context may be useful

Reply:
```suggestion
  mapLabels?: OptionT => React.Node,
```